src/raftstore/store/peer.rs

```diff
@@ -1372,6 +1375,7 @@ impl Peer {
         }
 
         metrics.keys_written += ctx.wb.count() as u64;
+        self.bytes_written += ctx.wb.data_size() as u64;
```

Comment:
Why not put it into RaftReadyMetrics？And we may need to count the size of entries written into rocksdb too.